src/nft/components/card/containers.tsx

```diff
@@ -95,7 +95,7 @@ const ActionButton = ({
     <StyledActionButton
       selected={isSelected}
       isDisabled={isDisabled}
-      onClick={(e: React.MouseEvent) => (isDisabled ? undefined : clickActionButton(e))}
+      onClick={(e) => (isDisabled ? undefined : clickActionButton(e))}
```

Comment:
why was this typing removed?

Reply:
It is unnecessary - the type is inferred from the event handler